docs/dev-tools/console/configuring-console.asciidoc

```diff
@@ -1,26 +0,0 @@
-[[configuring-console]]
-=== Configuring Console
-
-You can configure Console to your preferences.
-
-[float]
-==== Configuring settings
-
-*Settings* allows you to modify the font size and set the fileds for
-autocomplete.
-
-[role="screenshot"]
-image::dev-tools/console/images/console-settings.png["Console settings"]
```

Comment:
Any reason not to use this screenshot any more?

If we do decide to keep it, please note that it's outdated.

Reply:
Added it in and updated both images.